package.json

```diff
@@ -17,7 +17,8 @@
         "lint": "eslint --ext js,jsx src",
         "lint:file": "eslint",
         "lint:fix": "yarn lint -- --fix",
-        "lint:fix:dry": "yarn lint -- --fix-dry-run"
+        "lint:fix:dry": "yarn lint -- --fix-dry-run",
+        "lib:compile": "npx tsc"
```

Comment:
Is it possible to use `yarn` on invoke `tsc` rather than `npx`? This is in order to avoid depending on `npm` and `npx`.

Reply:
Didn't think of that. 
Done.